app/controllers/api/v1/vendors/items_controller.rb

```diff
@@ -0,0 +1,11 @@
+module Api
+  module V1
+    module Vendors
+      class ItemsController < Api::V1::ApplicationController
```

Comment:
I prefer this style to the API::V1::Vendors one as well.

Reply:
Interesting. I like that!